internal/api/main.go

```diff
@@ -32,7 +32,7 @@ const specFile = "/api/playbook-dispatcher/v1/openapi.json"
 const apiShutdownTimeout = 10 * time.Second
 
 func init() {
-	openapi3.DefineStringFormat("uuid", `^[a-f0-9]{8}-[a-f0-9]{4}-4[a-f0-9]{3}-[89aAbB][a-f0-9]{3}-[a-f0-9]{12}$`)
+	openapi3.DefineStringFormat("uuid", `^[a-f0-9]{8}-[a-f0-9]{4}-[45][a-f0-9]{3}-[89aAbB][a-f0-9]{3}-[a-f0-9]{12}$`)
```

Comment:
This redefines the format for all the other uuid fields. If we want to enable uuid v5 specifically for the sat_id field then we should define a new format type and use it only on that field.

Reply:
@jharting, I'm not sure I see a big reason to differential uuids for any of the different uses.  This would seems to add a little complexity that not I'm sure is warranted.  I would advocate for using this new regex for all uuid field validation, but want to make sure I am not overlooking something.

The only other thought is if we want to relax the uuid validation and support any version.  Meaning `[45][a-f0-9]{3}` becomes `[a-f0-9]{4}`.  I had suggested the current change to @tahmidefaz as it was a minimal change from the current logic.

Thoughts?

Reply:
If we want to change the validation logic for all uuid fields we can certainly do that. However, let's make sure that it does not happen as an unintended consequence of changing the `sat_id` but instead is an intentional change. For that I would expect at least the MR/commit message to reflect that, test coverage to go beyond just sat_id, etc.

Reply:
Since we want to unblock other teams I suggested to @tahmidefaz that we should wrap up this round of changes and look into a followup PR to add UUID v5 support for all fields, so we can spend additional time on it.